spec/available_injections.spec.js

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+const AVAILABLE_INJECTIONS = require("../src/data/injections");
+
+describe("AVAILABLE_INJECTIONS", () => {
+  it("contais at least one injection definition", () => {
+    expect(AVAILABLE_INJECTIONS.length).toBeGreaterThanOrEqual(1);
+  });
+
+  it("provide a unique ID for each injection", () => {
+    let ids = new Set();
+    let duplicates = new Set();
+
+    for (let injection of AVAILABLE_INJECTIONS) {
```

Comment:
extreeeeeeme nit!  feel free to use `const` instead of `let` in these kinds of loops (if you're hardcore enough to go that far :)